.github/pull_request_template.md

```
@@ -0,0 +1,27 @@
+## 補足
+<!-- レビュー時に注意してほしい点や補足事項があれば記載してください -->
+
+----
+
+<details>
+<summary>AIがレビューする際に遵守するべき指示</summary>
+
+- コメントは日本語で記述してください。
+- レビューの際は、変更点を明確に把握できるように、具体的なコードの変更内容を示してください。
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)